modules/swoop/swoop-bundle/main.tf

```diff
@@ -1,69 +1,74 @@
-resource "helm_release" "swoop_api" {
-  name = "swoop-api"
+resource "helm_release" "swoop_bundle" {
+  name = "swoop-bundle"
   namespace = var.namespace
   repository = "https://element84.github.io/filmdrop-k8s-helm-charts"
-  chart = "swoop-api"
-  version = var.swoop_api_version
+  chart = "swoop-bundle"
+  version = var.swoop_bundle_version
   atomic = true
   create_namespace = false
 
   set {
-    name  = "postgres.service.name"
+    name  = "swoop-api.enabled"
+    value = var.deploy_swoop_api
+  }
+
+  set {
+    name  = "swoop-api.postgres.service.name"
     value = "${var.custom_postgres_settings["postgres"]["service"]["name"]}.${var.postgres_namespace}"
   }
 
   set {
-    name  = "postgres.service.dbName"
+    name  = "swoop-api.postgres.service.dbName"
     value = var.custom_postgres_settings["postgres"]["service"]["dbName"]
   }
 
   set {
-    name  = "postgres.service.dbUser"
+    name  = "swoop-api.postgres.service.dbUser"
     value = var.custom_postgres_settings["postgres"]["service"]["dbUser"]
   }
 
   set {
-    name  = "postgres.service.dbPassword"
+    name  = "swoop-api.postgres.service.dbPassword"
     value = var.custom_postgres_settings["postgres"]["service"]["dbPassword"]
   }
 
   set {
-    name  = "postgres.service.sslMode"
+    name  = "swoop-api.postgres.service.sslMode"
     value = var.custom_postgres_settings["postgres"]["service"]["sslMode"]
   }
 
   set {
-    name  = "postgres.service.sslMode"
+    name  = "swoop-api.postgres.service.sslMode"
     value = var.custom_postgres_settings["postgres"]["service"]["sslMode"]
   }
 
   set {
-    name  = "postgres.service.authMethod"
+    name  = "swoop-api.postgres.service.authMethod"
     value = var.custom_postgres_settings["postgres"]["service"]["authMethod"]
   }
 
   set {
-    name  = "postgres.service.schemaVersionTable"
+    name  = "swoop-api.postgres.service.schemaVersionTable"
     value = var.custom_postgres_settings["postgres"]["deployment"]["schemaVersionTable"]
   }
 
   set {
-    name  = "minio.service.name"
+    name  = "swoop-api.minio.service.name"
     value = var.custom_minio_settings["minio"]["service"]["name"]
   }
 
   set {
-    name  = "minio.service.bucketName"
+    name  = "swoop-api.minio.service.bucketName"
     value = var.custom_minio_settings["minio"]["service"]["bucketName"]
   }
 
   set {
-    name  = "minio.service.accessKeyId"
+    name  = "swoop-api.minio.service.accessKeyId"
     value = var.custom_minio_settings["minio"]["service"]["accessKeyId"]
   }
 
   set {
-    name  = "minio.service.secretAccessKey"
+    name  = "swoop-api.minio.service.secretAccessKey"
     value = var.custom_minio_settings["minio"]["service"]["secretAccessKey"]
```

Comment:
if any of these are not found inside the `custom_postgres_settings` map, are the default values from the values.yaml used for these variables?

Reply:
yes, if we're not setting the value, the defualt values from the values.yaml in the helm chart would be used.